charts/cluster-scanner/templates/configmap.yaml

```
@@ -13,6 +13,9 @@ data:
   eve_enabled: {{ .Values.eveEnabled | quote}}
   eve_integration_enabled: {{ .Values.eveIntegrationEnabled | quote }}
   logging_level: {{ .Values.global.loggingLevel }}
+{{- if eq .Values.global.scannerMode "local" }}
+  local_registry_secrets: {{ include "cluster-scanner.runtimeStatusIntegrator.localCluster.localSecrets" . }}
```

Just for my understanding, what's the difference with the one below?
```suggestion
  local_registry_secrets: {{ - include "cluster-scanner.runtimeStatusIntegrator.localCluster.localSecrets" . }}
```